src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -122,6 +167,19 @@ class JooqTouchstoneRepository(
             mapEnum(record.status)
     )
 
+    fun mapDemographicStatisticType(records: List<Record>): DemographicStatisticType
+    {
+        val record = records[0]
```

Comment:
Will this always be one row? If so, better to use `single`. If there could be more rows, could you add a comment explaining why it's fine to just use the first one? Thanks.